internal/beater/beater.go

```diff
@@ -447,6 +454,11 @@ func (s *Runner) Run(ctx context.Context) error {
 		modelprocessor.NewDropUnsampled(false /* don't drop RUM unsampled transactions*/, func(i int64) {
 			transactionsDroppedCounter.Add(i)
 		}),
+
+		// Add a model processor that removes `event.received`, which is added by
+		// apm-data, but which we don't yet map.
```

Comment:
Outdated comment. Instead of "we don't yet map", we intentionally do not map it now.

What about also mentioning that it has to be after aggregation because aggregation adds it as well?